### PR TITLE
new blog bose

### PR DESCRIPTION
**Pull request by AbhishekBose**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
     "server": "hexo server"
   },
   "hexo": {
-    "version": "4.0.0"
```
Comment (rana-abhishek) [inline package.json:12]: Please don't change the hexo version as it causing the build to fail. Revert the changes done in package.json

File: package.json [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
+    "version": "4.2.0"
   },
   "dependencies": {
```
Comment (rana-abhishek) [inline package.json:14]: Same

File: source/_posts/Pyton-and-Kafka-Message-Passing-and-More.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+title: Python and Kafka: message passing and more
+date: 2020-05-03 14:50:04
+author: Abhishek Bose
```
Comment (rana-abhishek) [inline source/_posts/Pyton-and-Kafka-Message-Passing-and-More.md:4]: Please add category AI so that it can be shown under AI. `category: Android`

File: source/_posts/Pyton-and-Kafka-Message-Passing-and-More.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+category: AI
+tags:
```
Comment (Rajeev111974) [inline source/_posts/Pyton-and-Kafka-Message-Passing-and-More.md:6]: example test
